plugins/services/src/js/validators/MarathonAppValidators.js

```diff
@@ -137,6 +137,38 @@ const MarathonAppValidators = {
 
     // No errors
     return [];
+  },
+
+  /**
+   * This validator tests two fields:
+   *
+   * - container.volumes.X.containerPath should not contain '/'
+   *
+   * @param {Object} app - The data to validate
+   * @returns {Array} Returns an array with validation errors
+   */
+  containerVolmesPath(app) {
+    const volumes = findNestedPropertyInObject(app, 'container.volumes');
+    if (ValidatorUtil.isEmpty(volumes)) {
+      return [];
+    }
+
+    return volumes.reduce(function (memo, volume, index) {
+      const containerPath = volume.containerPath || '';
+
+      // Local volumes should not contain '/'
+      if (containerPath.indexOf('/') !== -1) {
+        memo.push({
+          path: ['container', 'volumes', index, 'containerPath'],
+          message: 'Should not contain "/"'
```

Comment:
What about other characters?
Could it be best to just say use alphanumeric characters?